beacon-chain/db/state_metrics.go

```diff
@@ -79,7 +79,7 @@ func reportStateMetrics(state *pb.BeaconState) {
 		// Track individual Validator's slashed epochs
 		validatorSlashedGauge.WithLabelValues(
 			strconv.Itoa(i), //Validator index
-		).Set(float64(v.SlashedEpoch))
```

Comment:
Temporary change to get build to pass, waiting for feedback.

Reply:
is there a way to tell which epoch validator got slashed?

Reply:
The spec has in `slash_validator`.

```
state.validator_registry[slashed_index].slashed = True
state.validator_registry[slashed_index].withdrawable_epoch = current_epoch + LATEST_SLASHED_EXIT_LENGTH
```

So maybe we can do `v.WithdrawableEpoch - LATEST_SLASHED_EXIT_LENGTH` to get the epoch they were slashed 👍